README.md

```diff
@@ -23,35 +23,28 @@ If you're on Windows, the easiest way to get started is to download the viewer f
 
 Or, you can browse a searchable list of samples on the ArcGIS for developers website:
 
-[![Link: Xamarin.Forms](https://img.shields.io/badge/Xamarin.Forms-3498db?style=flat-square&labelColor=gray&logo=Xamarin)](https://developers.arcgis.com/net/forms/sample-code/)
```

Comment:
We should add a line with a link to where MAUI will be - `https://developers.arcgis.com/net/maui/sample-code/`

Reply:
Created a line with a badge and link to the given page